src/Discovery/DiscoveryActionBar.tsx

```diff
@@ -188,6 +337,22 @@ const DiscoveryActionBar = (props: Props) => {
                 {(props.user.username) ? 'Open In Workspace' : 'Login to Open In Workspace'}
               </Button>
             </Popover>
+            {
+              props.downloadStatusMessage.active
+            && (
+              <Popup
```

Comment:
Just tried a quick run in QA, looks like after succeeded, the returned presigned URL text overflows the context area of this component

Suggest to replace it with https://ant.design/components/modal/ for better layout management, and you can also turn that presigned URL from plain text into HTML link

We are trying to swap out some our old UI components from Portal codes, `Popup` is one of them